layouts/alias.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="{{ site.Language.LanguageCode }}">
+  <head>
+    <title>{{ .Permalink }}</title>
+    <link rel="canonical" href="{{ .Permalink }}">
+    <meta name="robots" content="noindex">
+    <meta charset="utf-8">
+  </head>
+  {{/* If its the homepage, the target will end with just the /de/, all other will have a longer url */}}
+  {{ if hasSuffix .Permalink "/de/" }}
+  <noscript>
+    <meta http-equiv="refresh" content="0; url={{ .Permalink }}"/>
+  </noscript>
+  <script type="text/javascript">
+    let siteLangs = [{{ range site.Languages }} '{{ .LanguageCode }}', {{ end }}];
+
+    let findLang = () => {
+      for (i in navigator.languages) {
+        let lang = navigator.languages[i].split('-')[0];
+        if (siteLangs.includes(lang)) {
+          return lang;
+        }
+      }
+      return 'en';
+    }
+    window.location.replace("/" + findLang() + "/");
```

Comment:
I believe current best practises dictate using template literals instead of concatenation for this
```suggestion
    window.location.replace(`/${findLang()}/`);
```